### PR TITLE
Switch to GitHub Actions

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14'
+    - name: Run tests
+      run:
+      - travis_wait curl -L https://github.com/operator-framework/operator-sdk/releases/download/v0.18.2/operator-sdk-v0.18.2-$(uname
```
Comment (Fryguy) [inline .github/workflows/ci.yaml:26]: travis_wait is not here

Comment (Fryguy) [reply .github/workflows/ci.yaml:26]: Also can we move this into a bin/before_install for consistency with other repos?  before_install should be a shell script that checks for CI and installs any system packages and whatnot (Example: https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/bin/before_install).  before_install should be before the `Set up Ruby` step (and so should be a pure shell script (Example: https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/.github/workflows/ci.yaml#L30-L31)

Something like

```sh

Comment (bdunne) [reply .github/workflows/ci.yaml]: 👍🏻 Fixed

Comment (Fryguy) [reply .github/workflows/ci.yaml]: Can you do the bin/before_install script as well?

Comment (chessbyte) [reply .github/workflows/ci.yaml]: @bdunne ^^

Comment (bdunne) [reply .github/workflows/ci.yaml]: Done

**Comment by chessbyte** [comment]
@Fryguy seems like this ready for review

**Pull request merged**